src/wmtk/utils/tuple_sync.hpp

```diff
@@ -0,0 +1,4 @@
+#include <Mesh.hpp>
+
+// Map t_source from M_source to M_target
+Tuple map_tuple_between_meshes(const Mesh& M_source, const Mesh& M_target, const Tuple& t_source);
```

Comment:
namespace
i'd prefer `source_mesh` and `target_mesh` rather than mix a single character acronym